### DISCUSSION TITLE
Where can i find malilib for 1.20.5

### DISCUSSION THREAD

**Discussion by Fortbat** [maruohon/litematica#840] [Q&A] [answered]
I have  [()litematica](https://github.com/sakura-ryoko/litematica/releases/tag/1.20.5-sakura.2) for 1.20.5 but i don't know where to find malilib for version 1.20.5 , and it doesnt work without it

**Comment by maruohon** [comment, marked as answer]
It's also available in sakura-ryoko's forked repository. Click on the username to show their other repos. They have updates of all my client mods and also several other "related" mods.

**Reply by Fortbat** [reply]
thank you so much!
